src/BloomBrowserUI/bookEdit/BookPagesThumbnailList/BookPagesThumbnailList.htm

```diff
@@ -31,12 +31,36 @@
             jQuery('.gridItem').click(function () {
                 fireCSharpEvent("gridClick", $(this).attr('id'));
             });
+
+            setTimeout(loadNextThumbnail, 100);
```

Comment:
This pull request was done before I started working on PR #345. Without limiting the number of threads, the 100 ms timeout worked best.